README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Introduction
+
+My name is Aidan Coffey and I am from Long Island, NY. I am a 4th in the College majoring in Political Science and Public Policy. (*BA Spring 2020*). 
+
+## Politcal Science Background
+
+At UChicago, I have focused my study on American politics. I have supplanted that academic focus while organizing for the Missouri Democratic Party. I recommend you check out their website here! <https://missouridemocrats.org/>
+
+At the MO-DP, I..
+* knocked at least 100 doors a day
+* scheduled voluteers in the community to knock doors, make calls, and participate in our democracy
+* co-ran a team of interns of over 15 high school students
+* Worked 6 days a week from 8am-9pm!
```

Comment:
small thing, but just to keep it consistent I think the list elements should either all start out capitalized or none of them should